src/app/response-list.ts

```diff
@@ -0,0 +1,6 @@
+export interface ResponseList<T>{
```

Comment:
it depends on the project but most of the projects ( including ours )  follows pattern of naming model files with 'interface' in it 
i.e. response-list.interface.ts